bedrock/firefox/templates/firefox/desktop/index.html

```diff
@@ -21,7 +21,7 @@
   {% endif %}
 {% endblock %}
 
-{% block site_css %}
+{% block page_css %}
```

Comment:
The other desktop pages (`fast.html`, `trust.html`, `customize.html`) still use the `site_css` block. Should those be switch to `page_css` as well?